Rakefile

```diff
@@ -1,72 +1,50 @@
-require "json"
 require "pathname"
 
-CONFIG_FILE  = Pathname.new(__dir__).join(ENV["CNFFILE"] || "config.json")
 BUILD_ROOT   = Pathname.new(ENV["OBJDIR"] || "build")
 INSTALL_ROOT = Pathname.new(ENV["PREFIX"] || "/opt/ood")
+VENDOR_DIR   = Pathname.new(ENV["VENDOR_DIR"] || BUILD_ROOT.join("vendor").expand_path )
```

Comment:
This `VENDOR_DIR` is currently unnecessary. The bin/setup's of each app already have `VENDOR_BUNDLE_DIR` so whoever executes the rake task to build one or more of the apps can just set `VENDOR_BUNDLE_DIR=/path/to/vendor/bundle rake -mj build`. 

If we introduce another use for a shared vendor directory in the future (besides adding installed gem dependencies) this could be useful then.